src/test/fileUtils.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { parseCodacyrcFile } from "../fileUtils"
+import { deepEqual } from "assert"
+import { Codacyrc } from "../model/CodacyInput"
+
+describe("main", () => {
+  it("should parse a codacyrc file", () => {
```

Comment:
please have negative tests for checking the behavior with invalid `codacyrc` files

Reply:
✅Done